provd/rest/server/helpers/tenants.py

```diff
@@ -22,10 +22,6 @@ def autodetect(cls, request, tokens):
 
     @classmethod
     def from_headers(cls, request):
-        return cls.from_headers_one(request)
-
-    @classmethod
-    def from_headers_one(cls, request):
         tenant_uuid = request.getHeader('Wazo-Tenant')
         if not tenant_uuid:
             raise InvalidTenant()
```

Comment:
In `xivo-lib-python` you removed the condition that raises an InvalidTenant exception when there are multiple tenants. Is it supposed to be the same here?

Reply:
Fixed